### PR TITLE
Added set value to openvswitch_bridge

### PR DESCRIPTION
**Pull request by galexrt**
##### ISSUE TYPE
- Feature Pull Request
##### COMPONENT NAME

openvswitch_bridge
##### ANSIBLE VERSION

```
2.1.1.0
```
##### SUMMARY

This allow to set options for openvswitch bridges like the openvswitch_ports module already allows us to do.

### CODE REVIEW

**Comment by gregdek** [comment]
Thanks @galexrt. To the current maintainers, @stygstra please review according to guidelines (http://docs.ansible.com/ansible/developing_modules.html#module-checklist) and comment with text 'shipit', 'needs_revision' or 'close_me' as appropriate.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
Thanks @galexrt for this PR. Unfortunately, it is not mergeable in its current state due to merge conflicts. Please rebase your PR. When you are done, please comment with text 'ready_for_review' and we will put this PR back into review.

For help on how to do this cleanly please see http://docs.ansible.com/ansible/community.html#contributing-code-features-or-bugfixes

[This message brought to you by your friendly Ansibull-bot.]

File: network/openvswitch_bridge.py [hunk 1]
```diff
@@ -83,6 +102,7 @@ def __init__(self, module):
         self.state = module.params['state']
         self.timeout = module.params['timeout']
         self.fail_mode = module.params['fail_mode']
+        self.set_opt = module.params.get('set', None)
```
Comment (ghost) [inline network/openvswitch_bridge.py:131]: Could you add `'set'` to the `argument_spec` in `main()`?

``` python
            'set': {'required': False, 'default': None},
```

**Comment by galexrt** [comment]
ready_for_review

File: network/openvswitch_bridge.py [hunk 2]
```diff
@@ -89,6 +95,19 @@
...
 '''
 
+def _set_to_get(set_cmd, module):
+    """ Convert set command to get command and set value.
+    return tuple (get command, set value)
+    """
+
+    ##
+    # If set has option: then we want to truncate just before that.
+    set_cmd = truncate_before(set_cmd, " option:")
```
Comment (ghost) [inline network/openvswitch_bridge.py:114]: `truncate_before` appears to be undefined. Here's how it was defined in `openvswitch_port` if you want to copy that over:

``` python
def truncate_before(value, srch):
    """ Return content of str before the srch parameters. """

    before_index = value.find(srch)
    if (before_index >= 0):
        return value[:before_index]
    else:
        return value
```

**Review by ghost — CHANGES_REQUESTED** [review]
Thanks! A couple small changes and it'll be ready to go.

File: network/openvswitch_bridge.py (hunk 1 above)
Comment (galexrt) [reply network/openvswitch_bridge.py:131]: Done.

File: network/openvswitch_bridge.py (hunk 2 above)
Comment (galexrt) [reply network/openvswitch_bridge.py:114]: I copied it.

**Review by ghost — APPROVED** [review]

**Comment by ghost** [comment]
@galexrt Thanks!

@gregdek shipit

**Comment by gregdek** [comment]
Thanks again to @galexrt for this PR, and thanks @stygstra for reviewing. Marking for inclusion.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gundalow** [comment]
https://github.com/ansible/ansible/pull/19692 has been merged, so closing this

**Pull request closed without merging**